client/app/kubernetes/service/kubernetes-service.html

```diff
@@ -103,4 +115,5 @@
         </div>
     </div>
     <div data-ui-view="kubernetesResetView"></div>
+    <div data-ui-view="kubernetesUpdateView"></div>
```

Comment:
Two views one below the other ? Isn't the view a bit too much overloaded ?

Reply:
This has been replaced by a single view now :)